openedx/features/marketplace/constants.py

```diff
@@ -1,36 +1,100 @@
+    (ORGANIZATIONAL_PROBLEM_TYPE_DELIVERY_SERVICES_KEY, 'Delivery Services'),
+    (ORGANIZATIONAL_PROBLEM_TYPE_MENTORSHIP_KEY, 'Mentorship'),
+    (ORGANIZATIONAL_PROBLEM_TYPE_FUNDING_KEY, 'Funding'),
+    (ORGANIZATIONAL_PROBLEM_TYPE_HUMAN_RESOURCES_KEY, 'Human Resources'),
+)
+
+ORGANIZATIONAL_PROBLEM_CHOICES_VIEW = (
+    (ORGANIZATIONAL_PROBLEM_TYPE_HEALTH_CARE_SUPPLIES_KEY, 'Healthcare Supplies'),
+    (ORGANIZATIONAL_PROBLEM_TYPE_REMOTE_WORKING_TOOLS_KEY, 'Remote Working Tools'),
+    (ORGANIZATIONAL_PROBLEM_TYPE_ONLINE_TRAINING_KEY, 'Online Training'),
+    (ORGANIZATIONAL_PROBLEM_TYPE_DELIVERY_SERVICES_KEY, 'Delivery Services'),
+    (ORGANIZATIONAL_PROBLEM_TYPE_MENTORSHIP_KEY, 'Mentorship'),
+    (ORGANIZATIONAL_PROBLEM_TYPE_FUNDING_KEY, 'Funding'),
+    (ORGANIZATIONAL_PROBLEM_TYPE_HUMAN_RESOURCES_KEY, 'Human Resources'),
+    (ORGANIZATIONAL_PROBLEM_TYPE_OTHER_KEY, 'Other'),
+)
+
+ORGANIZATIONAL_SECTOR_HEALTH_AND_WELL_BEING = 'health-and-well-being'
+ORGANIZATIONAL_SECTOR_EDUCATION = 'education'
+ORGANIZATIONAL_SECTOR_GENDER_EQUALITY = 'gender-equality'
+ORGANIZATIONAL_SECTOR_SANITATION = 'sanitation'
+ORGANIZATIONAL_SECTOR_CLIMATE_CHANGES = 'climate-changes'
+ORGANIZATIONAL_SECTOR_CLEAN_ENERGY = 'clean-energy'
+ORGANIZATIONAL_SECTOR_ENVIRONMENTAL_CONSERVATION = 'environmental-conservation'
+ORGANIZATIONAL_SECTOR_WORK_AND_ECONOMIC_GROWTH = 'work-and-economic-growth'
+ORGANIZATIONAL_SECTOR_HUMAN_RIGHTS = 'human-rights'
+ORGANIZATIONAL_SECTOR_SOCIAL_JUSTICE = 'social-justice'
+ORGANIZATIONAL_SECTOR_ART_AND_CULTURE = 'art-and-culture'
+ORGANIZATIONAL_SECTOR_OTHER_KEY = 'other-sector'
+
 ORGANIZATION_SECTOR_CHOICES = (
-    ('health-and-well-being', _('Health & Well-being')),
-    ('education', _('Education')),
-    ('gender-equality', _('Gender Equality')),
-    ('sanitation', _('Sanitation')),
-    ('climate-changes', _('Climate Change')),
-    ('clean-energy', _('Clean Energy')),
-    ('environmental-conservation', _('Environmental Conservation')),
-    ('work-and-economic-growth', _('Work & Economic Growth')),
-    ('human-rights', _('Human Rights')),
-    ('social-justice', _('Social Justice')),
-    ('art-and-culture', _('Art & Culture'))
+    (ORGANIZATIONAL_SECTOR_HEALTH_AND_WELL_BEING, _('Health & Well-being')),
+    (ORGANIZATIONAL_SECTOR_EDUCATION, _('Education')),
+    (ORGANIZATIONAL_SECTOR_GENDER_EQUALITY, _('Gender Equality')),
+    (ORGANIZATIONAL_SECTOR_SANITATION, _('Sanitation')),
+    (ORGANIZATIONAL_SECTOR_CLIMATE_CHANGES, _('Climate Change')),
+    (ORGANIZATIONAL_SECTOR_CLEAN_ENERGY, _('Clean Energy')),
+    (ORGANIZATIONAL_SECTOR_ENVIRONMENTAL_CONSERVATION, _('Environmental Conservation')),
+    (ORGANIZATIONAL_SECTOR_WORK_AND_ECONOMIC_GROWTH, _('Work & Economic Growth')),
+    (ORGANIZATIONAL_SECTOR_HUMAN_RIGHTS, _('Human Rights')),
+    (ORGANIZATIONAL_SECTOR_SOCIAL_JUSTICE, _('Social Justice')),
+    (ORGANIZATIONAL_SECTOR_ART_AND_CULTURE, _('Art & Culture'))
```

Comment:
Why are we using translated strings here but not in `ORGANIZATIONAL_PROBLEM_CHOICES_VIEW`?